app/konnect/org-management/teams-roles-users.md

```diff
@@ -0,0 +1,195 @@
+
+* **Role:** Predefined access to a particular resource type (for example,
+  Services).
+
+When you create a Konnect account, you are automatically added to the Organization
+Admin team, which is one of the [predefined teams](/konnect/org-management/teams-reference)
+in Konnect. Predefined teams have sets of roles that can't be modified or
+deleted. You can add users to these teams, or create your own custom teams
+with any of the [supported roles](/konnect/org-management/roles-reference).
+
+Users can be part of any number of teams, and the roles gained from the teams
+are additive. For example, if you add a user to both the Service Developer and
+Portal Viewer teams, the user can create and manage Services
+through the Service Hub _and_ register applications through the Dev Portal.
+
+All users can view all teams and members of each team, but they can't view the
+team permissions.
+
+### Entity and role sharing
+
+An organization admin can share any role or entity with any user in the
+organization.
+
+Any user can also share Services or runtime groups that they have access to, with
+the same role or lesser.
+
+For example, say you have a Service Admin role:
+* You can share that Service with any other user through the Service Hub.
+* Since you have admin permissions, you can choose to share the Service any
+level of access: creator, deployer, viewer, etc.
+* The user that you share this Service with can then share the Service
+themselves.
+
+But what if you only have the Service Viewer role? In that case, you can only
+share the Service in read-only mode, as this is the lowest-level permission that
+you can have on the Service entity.
+
+You can [share any Service](/konnect/configure/servicehub/manage-services/#share-service)
+through the Service Hub, or
+[share any runtime group](/konnect/configure/runtime-manager/runtime-groups/manage/#share-runtime-group)
+through the Runtime Manager.
+
+### Create a new team
+
+1. From the left navigation menu in Konnect, open the
+![](/assets/images/icons/konnect/icn-organizations.svg){:.inline .no-image-expand .konnect-icon}
+ **Organization** link.
+2. Click **+ New Team**.
+3. Enter a team name and description. Both fields are required.
+4. Click **Save**.
+
+By default, your new team has the `Creator` role for all entities in the organization.
+Customize the roles through the team's configuration page.
+
+### Edit roles for a team
+
+You can edit the roles for any custom team. Any changes made to a team's roles
+are automatically applied to all team members.
+
+1. From the left navigation menu in Konnect, open the
+![](/assets/images/icons/konnect/icn-organizations.svg){:.inline .no-image-expand .konnect-icon}
+ **Organization** link.
+2. Choose a team from the list.
+
+    Teams with the `Predefined` label can't be edited or deleted.
+
+3. Choose an entity tab, then click **+ Add role(s)**.
+
+4. Enter the UUID of the element you want to assign, or enter `*` to target
```

Comment:
How do I get the UUID?

Reply:
This is a pre-helium iteration. Soon, this will be converted into a drop-down selector.

Reply:
The drop down will display a list of Services or RGs depending on what role you are adding. `*` will also be an option in the drop-down.

Reply:
To double-check: when we talk about service permissions, are we referring to Konnect services, gateway services, or both? Because I know how to find the UUID of a gateway service through the UI, but I don't think it's as easy to find the UUID of a Konnect service.